Wrong compiler version requirements in readme?

c++17 is required to build the project, isn't it? And the minimum versions of gcc and clang that support c++17 are 5 (for both compilers):

https://gcc.gnu.org/projects/cxx-status.html#cxx17
https://clang.llvm.org/cxx_status.html#cxx17

So, why do we have 4.8 and 3.4 in README.md??

```
[vagrant@localhost sql-parser]$ gcc --version
gcc (GCC) 4.8.5 20150623 (Red Hat 4.8.5-39)
Copyright (C) 2015 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

[vagrant@localhost sql-parser]$ make
g++ -std=c++1z -Wall -Werror -O3 -fPIC -c -o src/SQLParser.o src/SQLParser.cpp
g++: error: unrecognized command line option ‘-std=c++1z’
make: *** [src/SQLParser.o] Error 1
```

--- a/README.md
+++ b/README.md
@@ -14,7 +14,7 @@ In March 2015 we've also written a short paper outlining discussing some develop
 **Note:** You can also find a detailed usage description at this [blog post](http://torpedro.github.io/tech/c++/sql/parser/2016/02/27/c++-sql-parser.html).
 
 **Requirements:**
- * gcc 4.8+ (or clang 3.4+)
+ * gcc 5+ (or clang 5+)
 
 To use the SQL parser in your own projects you simply have to follow these few steps. The only requirement for is gcc 4.8+. Older versions of gcc/clang might also work, but are untested.
 

